tailor_image/create_image.py

```diff
@@ -141,10 +142,25 @@ def create_image(name: str, distribution: str, apt_repo: str, release_track: str
 
     elif build_type == 'ami':
         image_name = f'{organization}_{name}_{distribution}_ami_{release_label}'
+        # Get ami-id for base image
+        ec2 = boto3.client('ec2')
+        source_image_name = recipe[name]['source_ami'].replace('$distribution', distribution)
+        source_ami_id = None
+        images = ec2.describe_images(Owners=['self'])['Images']
```

Comment:
Is it possible to filter by name here? Looks like there's a filter argument

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Client.describe_images